frontend/web/components/pages/ImportPage.tsx

```diff
@@ -68,9 +68,9 @@ const ImportPage: FC<ImportPageType> = ({ projectId, projectName }) => {
       })
   }
 
-  const createImportLDProjects = (LDKey: string, projectId: string) => {
+  const createImportLDProjects = (LDKey: string, key: string, projectId: string) => {
     createLaunchDarklyProjectImport({
-      body: { project_key: 'default', token: LDKey },
+      body: { project_key: key, token: LDKey },
```

Comment:
What's the difference between a project key and a project id 🤔 ?

Reply:
Naming could be better here!

Reply:
I second that, but wanted a minimal diff for this one.